src/Client/app/(admin)/invite-candidates/page.tsx

```
@@ -18,255 +17,368 @@ import {
+              onChange={(e) => setEditedEmail(e.target.value)}
+              onBlur={() => {
+                handleEditEmail()
+                setEditingEmail(null)
+              }}
+              onKeyDown={(e) => {
+                if (e.key === 'Enter') {
+                  handleEditEmail()
+                  setEditingEmail(null)
+                }
+              }}
+              autoFocus
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)